python/README.md

```diff
@@ -0,0 +1,6 @@
+# Python
+
+Python code will adhere to the [PEP8](http://legacy.python.org/dev/peps/pep-0008/) Python coding standard.
```

Comment:
```suggestion
Python code should, as a minimum, adhere to the [PEP8](http://legacy.python.org/dev/peps/pep-0008/) Python coding standard.
We also encourage compliance with modern style recommendations, such as type annotations in [PEP484](https://peps.python.org/pep-0484/).
```
...?
@paddyroddy ?

Reply:
I'd be happy to add PEP484 / type annotations as a recommendation, previous material was just a verbatim copy from what was already in internal docs. I'll add this suggestion for now with the view that we can always iterate!

Reply:
Sounds good